packages/theming/win32-theme/src/__tests__/win32-theme.test.ts

```diff
@@ -0,0 +1,96 @@
+import { createOfficeTheme } from '../createOfficeTheme';
+import { createPartialOfficeTheme } from '../createPartialOfficeTheme';
+import { fallbackGetPalette, fallbackOfficeModule, getThemingModule } from '../NativeModule/index';
+import { createOfficeColorAliasTokens, createOfficeShadowAliasTokens } from '../createOfficeAliasTokens';
+import { createFontAliasTokens } from '../createFontAliasTokens';
+import { createBrandedThemeWithAlias, getCurrentBrandAliasTokens } from '../createBrandedThemeWithAlias';
+import { win32Typography } from '../getThemeTypography';
+import { ThemeOptions } from '@fluentui-react-native/theme-types';
+
+const themeOptions: ThemeOptions[][] = [
```

Comment:
win32 themes don't use light/dark/dynamic. We use Colorful/White/Dark Gray/Black/High Contrast

Reply:
I don't think it's necessarily valuable to test light/dark/dynamic separately for createOfficeTheme since createDefaultTheme unit tests should cover those cases

Reply:
Ahh that makes sense. I've updated it to just test w/ the light appearance.